src/pages/library/Datasets.js

```diff
@@ -435,11 +435,11 @@ const rareX = () => h(Participant, {
   sizeText: h(TooltipTrigger, { content: 'As of June 2022' }, [span('Participants: > 700')])
 }, [
   h(ButtonPrimary, {
-    'aria-label': 'Browse RareX data',
-    href: 'https://rare-x.org/xplore',
-    onClick: () => captureBrowseDataEvent('RARE-X'),
-    ...Utils.newTabLinkProps
-  }, ['Browse Data'])
+    'aria-label': 'Request access to RareX data',
+    tooltip: 'View access request form',
+    href: 'https://docs.google.com/forms/d/e/1FAIpQLScpqfZIXW53IJOiYz2RhASJzm7lZBYAjzkjJ67qFDERUpuDAQ/viewform',
+    onClick: () => captureBrowseDataEvent('RARE-X')
```

Comment:
Is this event still appropriate now that the link is to request access instead of view the data?

Reply:
Good question! I think it is still meaningful because to my knowledge, what we and the RARE-X folks will care about is still to know how many people went onto interfacing with RARE-X after viewing the library card.